core/src/test/scala/akka/persistence/cassandra/journal/CassandraJournalDeletionSpec.scala

```diff
@@ -84,6 +87,16 @@ class CassandraJournalDeletionSpec
     cassandra-query-journal-small-partition-size {
       write-plugin = cassandra-journal-small-partition-size
     }
+    
+    cassandra-journal-no-delete = ${cassandra-journal}
```

Comment:
change these to `$$` when https://github.com/akka/akka-persistence-cassandra/pull/451/files#diff-b97eaea2842359e4a03520b51f1235eeR62 has been merged